.travis.yml

```diff
@@ -4,6 +4,14 @@ sudo: false
 
 matrix:
   include:
+    - php: 5.3
+      env: WITH_LOWEST=true
+    - php: 5.3
+      env: WITH_HIGHEST=true
+    - php: 5.4
+      env: WITH_LOWEST=true
+    - php: 5.4
```

Comment:
According to https://travis-ci.org/codeclimate/php-test-reporter/jobs/191025428#L192-L199, it looks like we have an implicit version requirement of `>=5.5`.